static/src/javascripts/projects/common/modules/commercial/badges.js

```diff
@@ -5,168 +5,145 @@ define([
     'common/utils/$',
     'common/utils/config',
     'common/utils/template',
-    'common/utils/fastdom-idle',
+    'common/utils/fastdom-promise',
     'common/modules/commercial/dfp/add-slot',
     'common/modules/commercial/dfp/create-slot',
     'common/modules/commercial/commercial-features',
-    'text!common/views/commercial/badge.html',
-    'lodash/collections/map'
+    'text!common/views/commercial/badge.html'
 ], function (
     bonzo,
     qwery,
     Promise,
     $,
     config,
     template,
-    idleFastdom,
+    fastdom,
     addSlot,
     createSlot,
     commercialFeatures,
-    badgeTpl,
-    map
+    badgeTpl
 ) {
     var badgesConfig = {
-            sponsoredfeatures: {
-                count:      0,
-                header:     'Supported by',
-                namePrefix: 'sp'
-            },
-            'advertisement-features': {
-                count:      0,
-                header:     'Paid for by',
-                namePrefix: 'ad'
-            },
-            'foundation-features': {
-                count:      0,
-                header:     'Supported by',
-                namePrefix: 'fo'
-            }
-        },
-        addPreBadge  = function ($adSlot, header, sponsor) {
-            if (sponsor) {
-                $adSlot.append(template(
-                    badgeTpl,
-                    {
-                        header:  header,
-                        sponsor: sponsor
-                    }
-                ));
-            }
+        sponsoredfeatures: {
```

Comment:
Ah good to know, thanks! A minor point would then be why we have `sponsoredFeatures` and not `sponsored-features` but I don't think it will break the internet.